WYSIWYG-Quill/wysiwyg.html

```diff
@@ -0,0 +1,186 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="UTF-8">
+  <title>Wysiwyg</title>
+  <!-- Include jQuery library -->
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/KaTeX/0.7.1/katex.min.css" />
+  <link rel="stylesheet"
+    href="https://cdnjs.cloudflare.com/ajax/libs/highlight.js/9.12.0/styles/monokai-sublime.min.css" />
+  <link rel="stylesheet" href="https://amend.cz/wysiwyg/quill.snow.css" />
```

Comment:
Please use the official source defined in [documentation](https://quilljs.com/docs/download/)

```suggestion
  <link href="//cdn.quilljs.com/1.3.6/quill.snow.css" rel="stylesheet">
```